paper/paper.md

```diff
@@ -2,7 +2,7 @@
 title: 'Scientific and technical enhancement of bioinformatics software metadata using the Tools Ecosystem open infrastructure'
 title_short: 'Enhancement of bioinformatics software metadata'
```

Comment:
I think it would be better with a title that is more true to what we did. I'll give it a thought...